src/css/controls.less

```diff
@@ -14,6 +14,7 @@
 @import "controls/imports/nextup.less";
 @import "controls/imports/autostartmute.less";
 @import "controls/imports/settings-menu.less";
+@import "controls/imports/settings-menu-tizen.less";
```

Comment:
All tizen styles should be imported as part of a tizen module. The CSS for this less file should not be rolled up/loaded/injected with web player controls.

Reply:
I remember you mentioning this Rob, so I made a ticket to address this: JW8-11406

Reply:
Player controls inject controls.less with require
https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controls/controls.js#L21

Tizen controls should do the same with their own top level less file similar to this one.

Reply:
Agreed. I'll make sure to use that pattern when addressing JW8-11406

Reply:
This PR needs a rebase. We've merged https://github.com/jwplayer/jwplayer/pull/3757 so changing controls.less is no longer valid.